ssaEdge/eclipe-thingweb/thing-model/src/tm-helpers.ts

```
@@ -0,0 +1,712 @@
+    const data = _data as Record<string, unknown>;
+    if (Array.isArray(data["@type"])) {
+      const valid = data["@type"].filter((x) =>
+        x === "tm:ThingModel"
+      ).length > 0;
+      if (valid) {
+        return true;
+      }
+    } else if (data["@type"] === "tm:ThingModel") {
+      return true;
+    }
+    if (Object.keys(this.getThingModelRef(data)).length > 0) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)